R/misc.R

```diff
@@ -637,6 +637,27 @@ check_training_set <- function(x, rec, fresh) {
 }
 
 
+get_keep_original_cols <- function(object) {
```

Comment:
`embed` will need this too. Can you export and use `@keywords internal`?

Reply:
Done ✅